nixos/modules/installer/tools/nixos-option.sh

```diff
@@ -87,7 +92,6 @@ EOF
 ' <<EOF
 $result
 EOF
-      return 1;
```

Comment:
Why was this removed?

Reply:
good catch, that shouldn't be part of the patch :)

Reply:
Shouldn't the error still be propagated somehow? E.g. with an $error variable, which, when set, will return 1 at the very end, and intermediate failures will set it.